state-chain/chains/src/evm.rs

```diff
@@ -24,12 +24,21 @@ use sp_std::{convert::TryFrom, str, vec};
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct EvmCrypto;
 
+// TODO: Since they are different we can probably move this to its own
+// respective files (arb.rs and eth.rs)
```

Comment:
As explained in the comments, ARB & ETH have different gas consumptions. I assume this should be refactored in their own files.

Reply:
I agree separate modules would be better.